service/controller/aws/v2/version_bundle.go

```diff
@@ -10,17 +10,17 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "Cluster Operator",
-				Description: "TODO",
+				Component:   "cluster-operator",
+				Description: "Install chart-operator in kube-system namespace.",
```

Comment:
<3

Reply:
That should be `Installed` right? Isn't it always past tense and the verb is the very first thing?

Reply:
Yes it should, I'll fix.